src/app-layout/drawer/index.tsx

```diff
@@ -42,90 +24,143 @@ export interface DesktopDrawerProps {
 // * https://github.com/nvaccess/nvda/issues/5825
 // * https://github.com/nvaccess/nvda/issues/5247
 // * https://github.com/nvaccess/nvda/pull/8869 (reverted PR that was going to fix it)
+export const Drawer = React.forwardRef(
+  (
+    {
+      contentClassName,
+      toggleClassName,
+      closeClassName,
+      width,
+      type,
+      toggleRefs,
+      topOffset,
+      bottomOffset,
+      ariaLabels,
+      drawersAriaLabels,
+      children,
+      isOpen,
+      isMobile,
+      onToggle,
+      onClick,
+      onLoseFocus,
+      drawers,
+      resizeHandle,
+    }: DesktopDrawerProps,
+    ref: React.Ref<HTMLDivElement>
+  ) => {
+    const openButtonWrapperRef = useRef<HTMLElement | null>(null);
+    const { TagName, iconName, getLabels } = togglesConfig[type];
+    const { mainLabel, closeLabel, openLabel } = getLabels(ariaLabels);
+    const drawerContentWidthOpen = isMobile ? undefined : width;
+    const drawerContentWidth = isOpen ? drawerContentWidthOpen : undefined;
 
-export function Drawer({
-  contentClassName,
-  toggleClassName,
-  closeClassName,
-  width,
-  type,
-  toggleRefs,
-  topOffset,
-  bottomOffset,
-  ariaLabels,
-  children,
-  isOpen,
-  isMobile,
-  onToggle,
-  onClick,
-  onLoseFocus,
-}: DesktopDrawerProps) {
-  const { TagName, iconName, getLabels } = togglesConfig[type];
-  const { mainLabel, closeLabel, openLabel } = getLabels(ariaLabels);
-  const drawerContentWidthOpen = isMobile ? undefined : width;
-  const drawerContentWidth = isOpen ? drawerContentWidthOpen : undefined;
-  const openButtonWrapperRef = useRef<HTMLElement | null>(null);
+    const getDrawersLabels = (labels: DrawerItemAriaLabels = {}) => ({
+      drawerMainLabel: labels?.content,
+      drawerOpenLabel: labels?.triggerButton,
```

Comment:
This value is created but never used (also see the comment above how to eliminate `getDrawersLabels` completely)